pom.xml

```diff
@@ -250,6 +250,8 @@
                         <testTarget>${javaTestSource}</testTarget>
                         <source>${javaSource}</source>
                         <target>${javaSource}</target>
+                    	<fork>true</fork>
```

Comment:
I would like to see evidence that setting `fork=true` is a performance optimisation. I have rarely seen that and even on projects like https://github.com/jboss/jboss-parent-pom/blob/main/pom.xml#L314 for which the projects that use it have _many_ more source files fork is not used.